templates/CRM/Birthdays/Form/Settings.tpl

```diff
@@ -0,0 +1,196 @@
+{* HEADER *}
+
+<div class="crm-submit-buttons">
```

Comment:
Please use `<div class="crm-block crm-form-block">` containers for better implicit styling.